clang/include/clang/Basic/BuiltinsPPC.def

```diff
@@ -944,6 +944,9 @@ TARGET_BUILTIN(__builtin_pack_vector_int128, "V1LLLiULLiULLi", "", "vsx")
 // Set the floating point rounding mode
 BUILTIN(__builtin_setrnd, "di", "")
 
+// Barrier for instruction motion
```

Comment:
Can we add some comments here to emphasize that `__fence` will stop code motion for register based instructions?